packages/storage-adapter-local-storage/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
The file needs to exist so that the build process can overwrite it with generated docs.